samples/angular-todo/src/app/features/journey/form/form.component.ts

```diff
@@ -203,6 +218,28 @@ export class FormComponent implements OnInit {
    */
   handleStep(step?: FRStep) {
     this.step = step;
+
+    /**
+     * If the step contains TextOutputCallback, it means it has webAuthn callbacks so the sign-in button must be hidden as
+     * webAuthn will provide both biometric registration and biometric authentication log-in option buttons.
+     */
+    const textOutputCallbacks = step.getCallbacksOfType(CallbackType.TextOutputCallback);
+
+    if (textOutputCallbacks.length > 0) {
+      this.isShowSignIn = false;
+    } else {
+      this.isShowSignIn = true;
+    }
```

Comment:
What does `isShowSignIn` represent?

Reply:
Oh, I think I figured it out. Can we rename this button to be something more specific and accurate to its purpose? Something like `showSubmitButton`?

Reply:
yes, no problem. Can you talk it to ryan too? he'd asked me to name showSignIn(i made the change already locally)

Reply:
@ryanbas21: since Juan is using `isShowSignIn` as a hint to the UI regarding the display of a submit button, I'd like it to be more descriptive of intent. The sign in flow is not the only use case where the submit button may not be displayed, so naming it `showSignIn` is a bit misleading. Plus, a casual dev may not deduce what this does or doesn't do by reading this code alone.

Reply:
done